gov2/cloudwatch/README.md

```diff
@@ -127,4 +73,9 @@ If you want to see any log messages, enter:
 You should see some additional log messages.
 The last two lines should be similar to the previous output shown.
 
+## Additional resources
+
+- [AWS SDK for Go V3 Amazon IAM service reference](https://pkg.go.dev/github.com/aws/aws-sdk-go-v2/service/s3)
```

Comment:
The link text says IAM, but it links to S3, and it's in the CloudWatch README.

Reply:
Link fixed.